Leave/leave_interface.py

```diff
@@ -122,6 +122,27 @@ def AddRetroactiveLeaveToDB(member, message_id, start_date, end_date, leave_type
         leave_db.InsertLeave(member.id, message_id, leave_type, day, reason, "", leave_status, is_emergency, is_unpaid)
     return ("Retroactive leave was inserted successfully")
 
+def GetLeavesAcrossRange(start_date, end_date, member, include_reason):
+    start_date = ConvertToISO8601(start_date)
+    end_date = ConvertToISO8601(end_date)
+    embeds_to_send = []
+    leaves_group = []
+    leaves_group = GroupLeavesBy(leave_db.GetLeavesBetween(start_date, end_date, member), 'member_id')
+    for leaves_array in leaves_group:
+        leaves_sub_group = GroupLeavesBy(leaves_array, 'request_id')
+        embeds_to_send.append(UI.CreateMultipleLeavesEmbed(leaves_sub_group, include_reason))
+    return embeds_to_send
```

Comment:
This function definitely shouldn't be returning embeds. It should do exactly what it function name suggests—return leaves. We don't currently have leave objects, so returning them in whatever dictionary form is fine, but not in embeds. 

The output of this function should be passed into a function in the UI module to be converted into embeds. You can do these in the main flow in main.py

Reply:
Done